pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -70,6 +79,12 @@ lib.makeOverridable ({ # The kernel source tarball.
 # easy overrides to stdenv.hostPlatform.linux-kernel members
 , autoModules ? stdenv.hostPlatform.linux-kernel.autoModules
 , preferBuiltin ? stdenv.hostPlatform.linux-kernel.preferBuiltin or false
+, kernelTargets ?
+    if stdenv.hostPlatform.isAarch64 then
+      ["Image" "vmlinuz.efi"]
+    else
+      [stdenv.hostPlatform.linux-kernel.target]
+, installTargets ? lib.unique (map (target: defaultMapKernelToInstallTargets.${target} or "install") kernelTargets)
```

Comment:
I'm not sure how valuable it is to have a separate argument for `installTargets`, as I can't imagine a scenario where the `kernelTargets` passed wouldn't be directly correlated to `installTargets`. My understanding is the linux makefiles accept the kernel image targets as make targets already (e.g. `make bzImage` works directly, so something like `make Image vmlinuz.efi` should work as well)